### PR TITLE
Add PR Template

### PR DESCRIPTION
**Pull request by jordanstgodard**
## Jira Task
N/A

## What does this PR do?
* Adds a pull request template to make code reviews simple 👌 

## Why do we need this PR?
Documentation for code reviews is always beneficial. This template is an example that I've found works very well.

## How was this PR tested?
N/A

## What additional steps need to be conducted for releasing into QA or production?
N/A

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+## Jira Task
+https://jira.express-scripts.com/browse/PEPCLP-XXXX
```
Comment (devdowns) [inline pull_request_template.md:2]: We don't have this, can you convert to github issues instead

```suggestion
## Github Issue 
https://github.com/oburnett127/bankapp/issues/XXXX
```

**Review by devdowns — DISMISSED** [review]

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## Github Issue 
+https://github.com/oburnett127/bankapp/issues/XXXX
+
+## What does this PR do?
+
+
+## Why do we need this PR?
+
+
+## How was this PR tested?
```
Comment (oburnett127) [inline pull_request_template.md:10]: I think there is a better way to accomplish this

**Review by oburnett127 — APPROVED** [review]

File: pull_request_template.md (hunk 2 above)
Comment (devdowns) [reply pull_request_template.md:10]: Need to have a chat offline with @jordanstgodard

Comment (jordanstgodard) [reply pull_request_template.md:10]: Who needs testing? /s

**Pull request merged**